usermods/Ble2Json_v2/WLED_ESP32_4MB_1MB_FS.csv

```diff
@@ -0,0 +1,4 @@
+# Name,   Type, SubType, Offset,  Size, Flags
```

Comment:
Why this file?

Reply:
if really needed, then it would be better to add a new file into `tools/`, and reference it with `board_build.partitions = tools/WLED_ESP32_4MB_for_BLE.csv`
--> see my other comment about platformio_override.init.

Also, if you really need a different partitions table, then please also provide one for 8MB (typical flash size on -S3) and for 16MB (the future of ESP32).

Reply:
the program fits in the provided 8mb partition scheme... but I will move that file to tool... I just wanted to keep my stuff as isolated as possible.